src/liquid/components/ld-link/ld-link.tsx

```diff
@@ -79,6 +80,7 @@ export class LdLink implements ClonesAttributes {
         }
         rel={this.target === '_blank' ? 'noreferrer noopener' : undefined}
         disabled={this.disabled}
+        part="a"
```

Comment:
Can we please name it "link" or at least "anchor"? `::part(a)` may be a little confusing for a maintainer. - Also, shouldn't it also have the part "focusable" (and implement the InnerFocusable interface)?